Common/Orders/Fills/FillModel.cs

```diff
@@ -106,13 +113,202 @@ public virtual Fill Fill(FillModelParameters parameters)
             return new Fill(orderEvent);
         }
 
+        private bool CheckIfComboOrderIsReadyToFill(Order order)
```

Comment:
Could we replace the need of this method by using the `GroupOrderExtensions` maybe with some tweak?